lib/helpers/defaultTopicTrigger.js

```diff
@@ -12,8 +12,11 @@ const config = require('../../config/lib/helpers/defaultTopicTrigger');
 function fetchAll() {
   // TODO: Loop through all pages of query results - currently returns 100.
   return contentful.fetchByContentTypes(config.defaultTopicTriggerContentTypes)
-    .then(contentfulEntries => contentfulEntries.map(module.exports
-      .parseDefaultTopicTriggerFromContentfulEntry));
+    .then(contentfulEntries => Promise.all(contentfulEntries.map(module.exports
+      .parseDefaultTopicTriggerFromContentfulEntry)))
+    // Remove any draft entries that may have been parsed as null.
+    .then(defaultTopicTriggers => defaultTopicTriggers
+      .filter(defaultTopicTrigger => defaultTopicTrigger));
```

Comment:
It would make it a little more readable if we could move this filtering to a function instead.